src/utils.jl

```diff
@@ -301,7 +301,7 @@ end
     """ ->
     function gplot(sm::SimResult)
         N = length(sm.colnames)
-        figure()
+        Gaston.figure()
```

Comment:
I see you used Gaston. That must mean it still works (after your fix)! I wasn't sure.